src/app/components/navbar/navbar.component.ts

```diff
@@ -16,7 +16,7 @@ export class NavbarComponent implements OnInit, OnDestroy {
   lastName: string;
   profileImage: string;
   subscription = new Subscription();
-
+  userRole: string = 'BY';
```

Comment:
Quick question - why is this being hardcoded to `BY` here?

Reply:
@wangonya we agreed to _default_ this to a regular which is a buyer.